packages/react-native-client/plugin/src/withFishjamIOS.ts

```diff
@@ -0,0 +1,261 @@
+// ios-related code was mostly copied from OneSignal expo plugin: https://github.com/OneSignal/onesignal-expo-plugin/blob/main/onesignal/withOneSignalIos.ts
```

Comment:
WDYT about adding a little info in comment somewhere about what this plugin does?

Reply:
I added some additional comments on what each step does.